### PR TITLE
Add a page about using no business model

### PR DESCRIPTION
**Pull request by hennevogel**
This PR mainly serves as a basis for discussion of this aspect for the sfosc.

A large portion (the majority?) of Free and Open Source projects uses this *"business model"* including a significant amount of core GNU/Linux infrastructure and tools, that have arguably the longest track record of being sustainable.

How does this fit into the whole concept/basis of sfosc?

### CODE REVIEW

**Comment by hennevogel** [comment]
I also must say I severely dislike the name I gave it as I think most of those projects are run by computing professionals, just not in a professional context. Hobby has a bad ring to it in the framework of sustainability, yet I fail to come up with any other meaningful name...

**Comment by adamhjk** [comment]
First, thank you for the PR. I've been taking a bit of a break (I moved on from Chef after 10 years, so I've been playing a lot of video games and hanging around my house - it's a delight), so I'm way behind on looking at SFOSC.

I agree that "Hobby" isn't quite right, and that we're missing something about not having a "model" at all. Perhaps we should just call this section "None"? :)

Another approach would be to have the top-level business model page describe this - not every sustainable open source project needs or wants a business model. Similarly missing from the list is "big company exhaust", where we get things like envoy and (arguably) systemd.

**Comment by hennevogel** [comment]
> I've been playing a lot of video games and hanging around my house - it's a delight

Tell me more! NOT! 😤 \*envy\*

> so I'm way behind on looking at SFOSC.

Who isn't way behind on everything. Glad you found some cycles! 

> Perhaps we should just call this section "None"?

Oh yeah I like None. None first and describing that not everyone needs a model sounds cool. I'll get on it.  Stay stoked, don't forget to charge the controller :stuck_out_tongue_winking_eye:

**Comment by hennevogel** [comment]
> where we get things like envoy and (arguably) systemd.

I would guess from my experience with Kay, anti-(enterprise distro)-establishment (good old Hamburg school I guess) was definitely some motivator behind why systemd got some traction. Back then this part of the OS was entrenched and especially sysvinit was heavily fought over in LSB and other enterprise battlefields. Would you say that this is true @kaysievers / @poettering?

**Comment by davelab6** [comment]
We all live under capitalism as members of the working class - where we
must sell our time working for money - or members of a ownership class -
where we have coupons that entitle us to receive money without work. Where
there is a regulation of the work week, that money pays for a roof to sleep
8 hours, rest and live 8 hours, and if the former, work 8 hours for 5 days
a week.

The hobbyist model, where software goods are created in the "free time"
hours, is only possible when subsidised by money from wages or profits.

Rather than "None," I think "Leisure Time" is more descriptive.

Hope that helps :)

**Comment by hennevogel** [comment]
@adamhjk anything else I should do to get this merged?

**Comment by hennevogel** [comment]
> The hobbyist model, where software goods are created in the "free time" hours, is only possible when subsidised by money from wages or profits.

I think we are all very well aware of the BIAS this includes by now and that this should not be this way. But until we get there somehow, this is what it is. And it **is** big time...

> Rather than "None," I think "Leisure Time" is more descriptive.

"Leisure time" is no business model so "None" applies no?

File: content/business-models/none.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "None"
+date: 2019-01-09
+weight: 1
+---
+
+This is typically the ur-form of sustainable Free and Open Source Communities.
```
Comment (Beanow) [inline content/business-models/none.md]: Ur-form is not that commonly used a term, something more accessible like original form, or starting point is easier to read.

File: content/business-models/none.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+The business model is to *not have a business model*. That is okay, because
+not every sustainable open source project needs or wants a business model.
+
+In this case the project has set up means and rules of collaboration, has a
+formal membership, usually without a formal position of authority or any other
+hierarchy but does **not** deal with monetary contributions.
```
Comment (Beanow) [inline content/business-models/none.md]: "but does not deal with..." I think should be "_and_ does not..."

File: content/business-models/none.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+### Who uses it?
+
+* [sfosc](https://sfosc.org/)
+* [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
+* [zsh](http://zsh.org/)
+
+### When should it be used?
+
+In the early phase of projects when it's not clear that the idea/implementation
+is a viable option in the free software eco system. Or when it's clear from the
```
Comment (Beanow) [inline content/business-models/none.md]: "eco system" -> "ecosystem"

File: content/business-models/none.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+This is typically the ur-form of sustainable Free and Open Source Communities.
+The business model is to *not have a business model*. That is okay, because
+not every sustainable open source project needs or wants a business model.
+
+In this case the project has set up means and rules of collaboration, has a
+formal membership, usually without a formal position of authority or any other
+hierarchy but does **not** deal with monetary contributions.
+
+Sustainability in this non-model focuses on collaborator growth (quality and
+quantity) and shared responsibility to address it's biggest risk: contributor
```
Comment (NickKellett) [inline content/business-models/none.md]: spelling: _it's_ is short for "it is" and in this case should be _its_
`quantity) and shared responsibility to address its biggest risk: contributor`

File: content/business-models/none.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+desktop environment.
+
+### What kind of monetization is possible?
+
+Cash flow is avoided in this model because there is no legal entity behind the
+project (just individuals) that can handle the added complexity (control,
+reporting, taxes etc.) that monetization brings.
+
+The individuals involved cover the development costs for the project by
+donating their time. Infrastructure costs are avoided by utilizing free to use
```
Comment (NickKellett) [inline content/business-models/none.md]: Would it be useful to say "donating their time (and any trivial expenses)"?

Comment (Beanow) [reply content/business-models/none.md]: I think that's covered in the line below: "or by donations of goods by members (like paying for a domain name etc.)"

**Comment by Beanow** [comment]
@NickKellett thank you for helping out.
Looks like you also commented just on spelling/grammar. I'll have a go at resolving that.
Are you happy with the idea of the PR though? If so, I think we can merge after the small fixes.

File: content/business-models/none.md [hunk 6]
```diff
@@ -4,16 +4,16 @@ date: 2019-01-09
 weight: 1
 ---
 
-This is typically the ur-form of sustainable Free and Open Source Communities.
+This is typically the starting point for Sustainable Free and Open Source Communities.
```
Comment (hennevogel) [inline content/business-models/none.md]: "Starting point" sounds like you're supposed to leave this point and go on. While in reality most projects do not and don't have to be.

How about **archetype**?

> This is typically the archetype of Sustainable

Comment (Beanow) [reply content/business-models/none.md]: If we're moving further away from ur-from. Then maybe "common" instead of typical as well?

Comment (hennevogel) [reply content/business-models/none.md]: typically, commonly. [you say tomatoes I say tomatos](https://www.youtube.com/watch?v=q7a4z7zLKts&feature=youtu.be&t=59) :)

**Comment by Beanow** [comment]
@hennevogel, applied your suggestion. Using common, because I think it implies less that it makes sense as a default. I would rather people consider it in context of their own project :]

**Comment by Beanow** [comment]
@hennevogel if you're happy with this version, it has 2 reviews so I'll merge with your OK.

**Comment by Beanow** [comment]
I just noticed a thumbs up, so I'll take that as the OK :smile:
Thank you for seeing this PR through!

**Pull request merged**